api/v1beta1/openstacknetattachment_types.go

```diff
@@ -0,0 +1,146 @@
+/*
+
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1beta1
+
+import (
+	nmstateapi "github.com/nmstate/kubernetes-nmstate/api/shared"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// AttachType -
+type AttachType string
+
+const (
+	// AttachTypeBridge -
+	AttachTypeBridge AttachType = "bridge"
+	// AttachTypeSrIOV -
+	AttachTypeSrIOV AttachType = "sriov"
```

Comment:
nit: can we use Sriov everywhere (or SRIOV)

Reply:
donw